trade-finance-manager-ui/server/controllers/utilisation-reports/helpers/premium-payments-errors/add-payment-error.ts

```diff
@@ -0,0 +1,47 @@
+import { PREMIUM_PAYMENTS_TABLE_ERROR_HREF } from '../../../../constants/premium-payments-table-error-href';
+import { ADD_PAYMENT_ERROR_KEY } from '../../../../constants/premium-payment-tab-error-keys';
+import { AddPaymentErrorKey } from '../../../../types/premium-payments-tab-error-keys';
+import { ErrorSummaryViewModel } from '../../../../types/view-models';
+
+const { NO_FEE_RECORDS_SELECTED, DIFFERENT_STATUSES, DIFFERENT_PAYMENT_CURRENCIES, MULTIPLE_DOES_NOT_MATCH_SELECTED } = ADD_PAYMENT_ERROR_KEY;
+
+const addPaymentErrorMap: Record<AddPaymentErrorKey, ErrorSummaryViewModel> = {
```

Comment:
I'm not seeing any E2E coverage for these errors, unless i'm missing something?

Reply:
Looks like they were missed in the original implementation, have added 😄

Reply:
Amazing thanks 😄